src/main/resources/assets/neoforge/lang/en_us.json

```diff
@@ -154,8 +154,8 @@
   "neoforge.configuration.uitext.longstring": "This value is too long to be edited in the UI. Please edit it in the config file.",
   "neoforge.configuration.uitext.section": "%s...",
   "neoforge.configuration.uitext.sectiontext": "Edit",
-  "neoforge.configuration.uitext.breadcrumb.order": "%1$s%2$s%3$s",
-  "neoforge.configuration.uitext.breadcrumb.seperator": " > ",
+  "neoforge.configuration.uitext.breadcrumb.order": "%1$s %2$s %3$s",
+  "neoforge.configuration.uitext.breadcrumb.seperator": ">",
```

Comment:
I just realized: the proper spelling is "sep**a**rator", not "sep**e**rator". I think it's a minor enough change that we can include that in this PR.

Reply:
It's a new key anyway.

And really...English needs to start spelling its əs as ə, not as random other letters.